### PR TITLE
clarify localRepo is relative to $WORKSPACE

### PR DESCRIPTION
**Pull request by jtnord**
@alvarolobato  @reviewbybees

This bit me and it took me several times reading the doc - so make it more obvious.

### CODE REVIEW

**Comment by alvarolobato** [comment]
:bee:

**Comment by cyrille-leclerc** [comment]
🐝

File: src/main/resources/org/jenkinsci/plugins/pipeline/maven/WithMavenStep/help-mavenLocalRepo.html [hunk 1]
```diff
@@ -15,7 +15,9 @@
   <p>
   By using this option, Jenkins will tell Maven to use a custom path for the build as the local Maven repository by using <tt>-Dmaven.repo.local</tt>
   <br>
-  ie. <tt>$WORKSPACE/.repository</tt> if <tt>.repository</tt> value is specified.
+  If specified as a reltive path then this value well be resolved against the worksapce root and not the current working directory.
```
Comment (oleg-nenashev) [inline src/main/resources/org/jenkinsci/plugins/pipeline/maven/WithMavenStep/help-mavenLocalRepo.html]: Typo: "worksapce"

Comment (alvarolobato) [inline src/main/resources/org/jenkinsci/plugins/pipeline/maven/WithMavenStep/help-mavenLocalRepo.html]: and "reltive" is also wrong.

**Comment by alvarolobato** [comment]
@jtnord  can you, please, correct those two typos so I can merge this?

**Comment by cyrille-leclerc** [comment]
@jtnord can we close this PR as we have merged #21 ?

**Pull request closed without merging**